tests/indexing_test.py

```diff
@@ -89,3 +89,8 @@ def test_bigint_indexing_preserves_max_bits(self):
         a = ak.arange(2**200 - 1, 2**200 + 11, max_bits=max_bits)
         self.assertEqual(max_bits, a[ak.arange(10)].max_bits)
         self.assertEqual(max_bits, a[:].max_bits)
+
+    def test_handling_bigint_max_bits(self):
+        a = ak.arange(2**200 - 1, 2**200 + 11)
+        a[:] = ak.array(a, dtype=ak.bigint, max_bits=3)
+        self.assertListEqual([7, 0, 1, 2, 3, 4, 5, 6, 7, 0, 1, 2], a.to_list())
```

Comment:
same deal this test also needs to be updated

Reply:
Updated